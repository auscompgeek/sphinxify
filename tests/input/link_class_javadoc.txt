  /**
   * Use this method to define your button->command mappings. Buttons can be created by
   * instantiating a {@link GenericHID} or one of its subclasses ({@link
   * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then calling passing it to a
   * {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
   */
