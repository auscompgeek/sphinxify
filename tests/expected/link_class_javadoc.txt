Use this method to define your button->command mappings. Buttons can be created by
instantiating a :class:`.GenericHID` or one of its subclasses ({@link
edu.wpi.first.wpilibj.Joystick} or :class:`.XboxController`), and then calling passing it to a
:class:`.JoystickButton`.
